### DISCUSSION TITLE
Configure JKBMS over Venus OS / serialbattery driver

### DISCUSSION THREAD

**Discussion by herrep** [Louisvdw/dbus-serialbattery#191] [Q&A] [answered]
Hi,

The serialbattery driver reads out all relevant information from JKBMS via its RS485 interface.

I wonder whether it is possible or planned that BMS configuration can be done from the serialbattery driver via RS485. There are already certain settings to be defined for the BMS in the driver. At present, I use the JKBMS App for my smartphone. But this has two drawbacks.

At first, it would be very convenient to have all settings in one place (which avoids inconsistencies). At second, the dependency to an app and the smartphone OS is removed. I have concerns that in a few years the old apps are no longer maintained and do not work with current smartphone OS.

Best regards,
Peter

**Comment by Louisvdw** [comment, marked as answer]
This would be possible, but it is not something that is planned in the close future. 
I had similar idea, but there is already so much tickets just with the reading of data and getting new batteries added that I don't know when there will be time. This adds a lot of work per BMS.

**Reply by herrep** [reply]
Thank you very much for your commitment to this great project!

**Reply by dominikfe** [reply]
JK BMS transports a lot more information via bluetooth. In addition to dbus-serialbattery via usb I have connected to jk-bms via https://github.com/syssi/esphome-jk-bms. It reads out all values available in the iOS app and in the meantime you can even configure a lot of settings through it.
